docs/standard/containerized-lifecycle-architecture/docker-devops-workflow/docker-application-outer-loop-devops-workflow.md

```diff
@@ -43,191 +43,169 @@ When using Docker for the deployment, the "final artifacts" to be deployed are D
 
-The Docker extension can use service endpoints for Docker hosts and for container or image registries. The tasks default to using a local Docker host if available (this currently requires a custom Azure DevOps Services agent); otherwise, they require that you provide a Docker host connection. Actions that depend on being authenticated with a Docker registry, such as pushing an image, require that you provide a Docker registry connection.
+You can use these templates and tasks to construct your CI/CD artifacts to Build / Test and Deploy in Azure Service Fabric, Azure Kubernetes Service, and similar offerings.
 
-The Azure DevOps Services Docker extension installs the following components in your Azure DevOps Services account:
-
--   A service endpoint for connecting to a Docker registry
-
--   A service endpoint for connecting to a Docker Container Host
-
--   A Docker task to do the following:
-
--   Build an image
-
--   Push an image or a repository to a registry
-
--   Run an image in a container
-
--   Run a Docker command
-
--   A Docker Compose task to run a Docker Compose command
-
-With these Azure DevOps Services tasks, a build Linux-Docker Host/VM provisioned in Azure and your preferred Docker registry (Azure Container Registry, Docker Hub, private Docker DTR, or any other Docker registry) you can assemble your Docker CI pipeline in a very consistent way.
+With these Visual Studio Team Services tasks, a build Linux-Docker Host/VM provisioned in Azure and your preferred Docker registry (Azure Container Registry, Docker Hub, private Docker DTR, or any other Docker registry) you can assemble your Docker CI pipeline in a very consistent way.
 
 ***Requirements:***
 
--   Azure DevOps Services, or for on-premises installations, Team Foundation Server 2015 Update 3 or later.
-
--   An Azure DevOps Services agent that has the Docker binaries.
-
-An easy way to create one of these is to use Docker to run a container based on the Azure DevOps Services agent Docker image.
-
-**More info** To read more about assembling a Azure DevOps Services Docker CI pipeline and to view walkthroughs, visit the following sites:
+- Azure DevOps Services, or for on-premises installations, Team Foundation Server 2015 Update 3 or later.
 
-Running an Azure DevOps Services agent as a Docker container: [https://hub.docker.com/r/\
-microsoft/vsts-agent/](https://hub.docker.com/r/microsoft/vsts-agent/)
+- An Azure DevOps Services agent that has the Docker binaries.
 
-Azure DevOps Services Docker extension: <https://aka.ms/vstsdockerextension>
+  An easy way to create one of these agents is to use Docker to run a container based on the Azure DevOps Services agent Docker image.
 
-Building .NET Core Linux Docker images with Azure DevOps Services: <https://blogs.msdn.microsoft.com/stevelasker/2016/06/13/building-net-core-linux-docker-images-with-visual-studio-team-services/>
+> [!INFORMATION]
```

Comment:
INFORMATION is not one of the note types. Should this be a !NOTE instead?

https://github.com/dotnet/docs/blob/master/styleguide/template.md#alerts

Reply:
Yep, I just told @rpetrusha I'll include this as one more Regex change.